### PR TITLE
redesign page, add functionality of uploader, create adaptivity

### PR DESCRIPTION
**Pull request by Yevhendzyuba**
image
image
image

### CODE REVIEW

File: src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx [hunk 1]
```diff
@@ -61,7 +79,18 @@ const AddPhotoStep = ({ btnsBox, stepLabel }) => {
             validationData={ validationData }
           />
         </Box>
-
+        { (isMobile || isTablet ) && (
```
Comment (Tolik170) [inline src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx:82]: As i see, it is the same like above, you can move this box to variable and use it in different screen sizes

File: src/containers/tutor-home-page/add-photo-step/AddPhotoStep.style.js [hunk 2]
```diff
@@ -4,39 +4,48 @@ export const style = {
...
-    maxWidth: '580px',
+    maxWidth: '440px',
     width: '100%',
     aspectRatio: '1',
-    border: 'dashed',
+    border: '2px dashed',
     borderColor: 'primary.200',
-    borderRadius: '20px'
+    borderRadius: '20px',
+    mt: { xs: '20px', sm: '20px', md: '0px' }
```
Comment (Tolik170) [inline src/containers/tutor-home-page/add-photo-step/AddPhotoStep.style.js:21]: you can remove sm

File: src/containers/tutor-home-page/add-photo-step/AddPhotoStep.style.js [hunk 3]
```diff
@@ -4,39 +4,48 @@ export const style = {
...
   },
   img: {
     width: '100%',
-    maxWidth: '580px',
-    borderRadius: '20px'
+    borderRadius: '20px',
+    mt: { xs: '20px', sm: '20px', md: '0px' }
```
Comment (Tolik170) [inline src/containers/tutor-home-page/add-photo-step/AddPhotoStep.style.js:26]: same here

File: src/components/file-uploader/FileUploader.jsx [hunk 4]
```diff
@@ -33,30 +33,28 @@ const FileUploader = ({ buttonText, emitter, initialState = [], initialError = '
...
-      <Box
-        data-testid='drop'
-        onDragLeave={ dragLeave }
-        onDragOver={ dragStart }
-        onDragStart={ dragStart }
-        onDrop={ dragDrop }
-        sx={ [styles.root, isDrag && styles.rootDrag] }
-      >
-        { initialState.length > 0 && (<List sx={ { width: '100%' } }>
+      <Box data-testid='upload' sx={ styles.root }>
```
Comment (Tolik170) [inline src/components/file-uploader/FileUploader.jsx:49]: you did not use this data-testid, so you can remove it

**Review by Tolik170 — APPROVED** [review]

**Review by Roman-Peretiatko — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**